### PR TITLE
Add a `is-checked` class to element where the input is checked to ease customisation

### LINKED ISSUE #140: Customise label when input is inside of it and checked
This stackoverflow resume the issue at hand.
http://stackoverflow.com/questions/13316966/css-checkboxes-radio-buttons-when-input-is-inside-label

Currently, for `radios` and `checklist` (at least), it is impossible to do some customization to a checked fields.
For radios, ths current structure is like that:
```html
<div class="radio-list">
    <label>
        <input name="869beac8" value="Yes" type="radio"> Yes
    </label>
    <label>
        <input name="869beac8" value="No" type="radio"> No
    </label>
</div>
```
I want to change `label` appearance. I want to put a border around `label` when `input` is `checked`.
Right, now it is impossible.
There is two solutions as far as I can tell:
1. change the structure a bit
2. add some JavaScript behavior

here are the details
#### 1. change the structure
For example, if we add `<span>` around the text content of the label, we could do that:
```html
<div class="radio-list">
    <label>
        <input name="869beac8" value="Yes" type="radio">
        <span>Yes</span>
    </label>
    <label>
        <input name="869beac8" value="No" type="radio">
        <span>No</span>
    </label>
</div>
```
and then in css:
```css
input[type=radio]:checked + span {
    color: green;
}
```
#### 2. JavaScript behavior
While I could pretend to have a single box around both element with the CSS, it is a bit clunky and weak.
What could be better would be to change a class in the parent element.
```html
<!-- At first it's like that -->
<div class="radio-list">
    <label>
        <input name="869beac8" value="Yes" type="radio"> Yes
    </label>
    <label>
        <input name="869beac8" value="No" type="radio"> No
    </label>
</div>

<!-- Then I click "yes" -->
<div class="radio-list">
    <label class="is-checked"><!-- class for the state -->
        <input name="869beac8" value="Yes" type="radio" checked> Yes
    </label>
    <label>
        <input name="869beac8" value="No" type="radio"> No
    </label>
</div>
```
All I have to do is use this class, and I have much more liberties:
```css
label.is-checked {
    border: 2px solid green;
}
```
I'm more for the second option and I can do the PR if needed.
What are your thought ?

### PR DESCRIPTION
**Pull request by lionel-bijaoui**
close #140

### CODE REVIEW

**Comment by icebob** [comment]
Could you add tests, which checks the new classes?

File: test/unit/specs/fields/fieldRadios.spec.js [hunk 1]
```diff
@@ -64,31 +66,77 @@ describe("FieldRadios.vue", function() {
...
+			expect(labelList[1].classList.contains("is-checked")).to.be.true;
+			expect(labelList[2].classList.contains("is-checked")).to.be.false;
+			expect(labelList[3].classList.contains("is-checked")).to.be.false;
+			expect(labelList[4].classList.contains("is-checked")).to.be.false;
+			expect(labelList[5].classList.contains("is-checked")).to.be.false;
+			expect(labelList[6].classList.contains("is-checked")).to.be.false;
+		});
+
+
+		describe("test values reactivity to changes", () => {
```
Comment (lionel-bijaoui) [inline test/unit/specs/fields/fieldRadios.spec.js]: I have grouped the test a little bit, is this okay, and can I do the same thing in `checklist`'s tests ?

File: test/unit/specs/fields/fieldRadios.spec.js [hunk 2]
```diff
@@ -64,31 +66,77 @@ describe("FieldRadios.vue", function() {
...
+					expect(model.skills).to.be.equal("HTML5");
+					done();
+				});
+			});
+		});
 
-			vm.$nextTick( () => {
-				expect(model.skills).to.be.equal("HTML5");
-				done();
+		describe("test 'is-checked' class attribution reactivity to changes", () => {
```
Comment (lionel-bijaoui) [inline test/unit/specs/fields/fieldRadios.spec.js]: Same thing, grouping test for better readability

**Comment by lionel-bijaoui** [comment]
@icebob can you validate what I did, this is blocking my progress on this PR. Thanks !

**Comment by icebob** [comment]
I checked & approved your changes, just I thought you will do [this](https://github.com/icebob/vue-form-generator/pull/141#discussion_r104724544) grouping on checklist too in this PR.

**Comment by lionel-bijaoui** [comment]
@icebob Is everything OK ? I can merge ?

**Comment by icebob** [comment]
Ok, mergeable.

**Pull request merged**